desktop_version/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ endif()
 if(${CMAKE_VERSION} VERSION_LESS "3.1.3")
     message(WARNING "Your CMake version is too old; set -std=c90 -std=c++98 yourself!")
```

Comment:
```suggestion
    message(WARNING "Your CMake version is too old; set -std=c99 -std=c++98 yourself!")
```

Reply:
Does this mean we no longer have to worry about putting variable declarations at the top of a scope? That aspect of C90 is basically the sole reason why I don't want to see VVVVVV moving to C, so I've been really hoping for this change to happen.

Reply:
Just looked into it a bit... as far as I can tell, FAudio is C99 solely for use of `inline` and `restrict`, which can be easily worked around with macros and such. However my quick build on N00by's fork fails to find FAudio at link time, so maybe there's more problems past that or something.

Reply:
Also the comments in `stb.h` and `stb_vorbis.h` are C99. But after removing those with `cpp -P`, it compiles, without extensions.

Reply:
Though I'd be fine with being on C99 too.

Reply:
FAudio really needs both of those to work optimally, so it looks like FAudio's making the final call here!

Reply:
Alternatively, we could also enable C99 for FAudio, just like we enable C extensions for PhysFS...

Reply:
I mean, if we're able to use C99, I don't see why we should take the extra effort to apply different standards versions to different parts of the code, when we can just use C99 across the board... Plus, I really think we should drop C90 (and the particular C90-ism I mentioned).